content/guides/contribute/design.mdx

```diff
@@ -1,70 +1,56 @@
 ---
 title: Design
-description: These are some guidelines you should always keep in mind when creating Primer design patterns.
+description: Guidelines for building design patterns using Primer.
 ---
 
-The key goals are to create flexible, robust, and useful patterns that advance Primer's incremental innovation, while building upon an existing set of solid foundations.
+Design flexible, robust, and repeatable patterns that build on a set of solid foundations.
 
 ## Design system foundations
 
-Primer is built on strong foundations that should remain stable. Become familiar with guidelines on typography, color, spacing, and layout.
+Primer is the stable trusted layer. At its foundations are typography, color, spacing, and layout.
 
 ➡️ [Primer design guidelines](/)
 
 ## Upcoming patterns and proposals
 
-It's important to know what's coming into Primer, to avoid duplicated efforts and to accommodate upcoming changes to elements that may interact with what you're planning to work on.
+To avoid duplicated efforts and to accommodate upcoming changes, keep up with the Primer Roadmap.
 
-Check what is currently being designed and built, as this may introduce new concepts and variations to existing patterns.
-
-➡️ [Primer backlog](https://github.com/github/primer/projects/1?fullscreen=true) (GitHub staff only)
+➡️ [Primer Roadmap](https://github.com/orgs/github/projects/2759/views/29) (GitHub staff only)
```

Comment:
This link seems like it can become outdated _very_ easily. Any ideas how we can protect against that?